packages/mobile/src/harmony-native/foundations/typography/Text.tsx

```diff
@@ -116,21 +28,33 @@ export const Text = (props: TextProps) => {
     color: colorProp = 'default',
     ...other
   } = props
-  const styles = useStyles()
   const theme = useTheme()
   // TODO: make heading a proper gradient
   const color =
     colorProp === 'heading'
       ? theme.color.secondary.secondary
       : theme.color.textIcon[colorProp]
 
-  const styleKey = `${variant}-${size}-${strength}`
+  const variantStyles = variantStylesMap[variant]
+  // const variantStyles = variantStylesMap.label
+
+  const textStyles = useMemo((): TextStyle => {
```

Comment:
lets use the css prop instead of styles. this will do the `Stylesheet.create` for us, which is more performant